### ISSUE TITLE
Wrap TypeLiteralNode

### ISSUE THREAD

**Issue by kjpou1** [dsherret/ts-simple-ast#259]
Would open up the possibility to navigate the ts.TypeLiteralNode <=> ts-simple-ast without casting.

Contrived Example:

```
  const typeNode = node.getTypeNode();
  const typeLiteralNode = <ts.TypeLiteralNode>typeNode.compilerNode;

  const members = typeLiteralNode.members;

  members.forEach(member => {
    console.log(sast.createWrappedNode(member).getKindName());
  });

```

Example Definition:

```
interface Faux {
}

declare var Faux: {
    prototype: Faux;
    new(): Faux;
};

```

**Comment by dsherret** [comment]
This is now available in v8.1.

**Comment by kjpou1** [comment]
Tested and works.  Thanks
